### PR TITLE
Only escape commandline arguments on macOS take 2

### PR DESCRIPTION
**Pull request by Jesse-Bakker**
Fixed the signature of the macos version

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -68,6 +67,7 @@ version = "0.52.0"
 [target.'cfg(target_os = "macos")'.dependencies]
 cocoa = "0.24.0"
 objc = "0.2.7"
+shlex = "1.1.0"
```
Comment (williamspatrick) [inline Cargo.toml:70]: nit: appears to be a DOS-style new line here?

Locally I see this.
```
+shlex = "1.1.0"^M
```

**Review by williamspatrick — APPROVED** [review]
I tested this commit and it fixes #1659

File: Cargo.toml (hunk 1 above)
Comment (Jesse-Bakker) [reply Cargo.toml:70]: Weird, because I'm on linux and I don't see it

Comment (Jesse-Bakker) [reply Cargo.toml:70]: I see it now. The whole file seems to have DOS line endings when I open it with `set ffs=unix` :frowning:, but the diff only shows an added one on that line. I have removed it from that line.

**Comment by MultisampledNight** [comment]
Thank you!

**Pull request merged**